packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -494,12 +495,23 @@ async function handler(req: NextApiRequest & { userId?: number | undefined }) {
       language: { translate: tAttendees, locale: language ?? "en" },
     },
   ];
-  const guests = (reqBody.guests || []).map((guest) => ({
-    email: guest,
-    name: "",
-    timeZone: reqBody.timeZone,
-    language: { translate: tGuests, locale: "en" },
-  }));
+
+  const guests = (reqBody.guests || []).reduce((guestArray, guest) => {
+    // If it's a team event, remove the team member from guests
```

Comment:
On rescheduling, team members are stored as guests on the booking page. We need to remove them as guests and add them back as team members